types/serverless/index.d.ts

```diff
@@ -83,6 +83,15 @@ declare class Serverless {
     version: string;
 
     resources: AwsProvider.Resources;
+
+    configSchemaHandler: {
+        defineCustomProperties(props: any): any;
+        defineFunctionEvent(provider: string, event: string, props: any): any;
+        defineFunctionEventProperties(provider: string, existingEvent: string, props: any): any;
+        defineFunctionProperties(provider: string, props: any): any;
+        defineProvider(provider: string, props: any): any;
```

Comment:
In `lib/classes/ConfigSchemaHandler/index.js`, `defineProvider` is defined as requiring bewteen 1 and 2 arguments (`options` is optional and defaults to an empty object). The shape of the options object can be defined in later stage, but prefer the use of `Record` and `unknown` rather than `any`. The function is not returning anything, so use `void`
 ```suggestion
        defineProvider(provider: string, options?: Record<string, unknown>): void;
```